samples/drivers/kscan_ft5336/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
+  name: KSCAN ft5336 driver sample
+tests:
+  sample.drivers.kscan_ft5336:
+    tags: drivers
+    harness: console
+    harness_config:
+        type: multi_line
+        ordered: true
+        regex:
+            - "kb data(.*)"
+    depends_on: kscan
```

Comment:
You'll want to add `kscan` to the supported list in `boards/arm/stm32f746g_disco/stm32f746g_disco.yaml` to ensure this sample builds for this board in CI. Do it in commit `boards: Configure ft5336 touch panel on stm32f746g_disco board`

Reply:
I changed it to kscan_ft5336 so it doesn't run the other kscan sample build.